src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts

```diff
@@ -59,20 +64,23 @@ export abstract class SimpleFindWidget extends Widget {
 					new RegExp(value);
 					return null;
 				} catch (e) {
-					this.foundMatch = false;
-					this.updateButtons(this.foundMatch);
+					this._foundMatch = false;
+					this.updateButtons(this._foundMatch);
```

Comment:
TODO: combine `updateButtons` with `updateMatchCount` and get rid of `this._findMatch`